components/txn_types/src/write.rs

```diff
@@ -179,6 +224,9 @@ impl WriteRef<'_> {
             b.push(v.len() as u8);
             b.extend_from_slice(v);
         }
+        if self.has_overlay_rollback {
+            b.push(FLAG_OVERLAY_ROLLBACK);
```

Comment:
Should we change the capacity calculation?

Reply:
Oh. let me see.

Reply:
yes but we want it be always sufficient. It's actually the maximum possible length.